globus_automate_client/cli/actions.py

```diff
@@ -28,14 +69,22 @@ def action_introspect(
         callback=url_validator_callback,
     ),
     verbose: bool = verbosity_option,
+    output_format: ActionOutputFormat = typer.Option(
+        ActionOutputFormat.json,
+        "--format",
+        "-f",
+        help="Output display format.",
+        case_sensitive=False,
+        show_default=True,
+    ),
 ):
     """
     Introspect an Action Provider's schema.
     """
     ac = create_action_client(action_url, action_scope)
     if ac is not None:
         result = ac.introspect()
-        format_and_echo(result, verbose=verbose)
+        format_and_echo(result, DUMPERS[output_format], verbose=verbose)
```

Comment:
So, if `dump` were moved into `ActionOutputFormat` this could become `output_format.dump(result)` maybe?